src/main/java/io/neonbee/data/internal/DataContextImpl.java

```diff
@@ -241,7 +325,8 @@ public static String encodeContextToString(DataContext context) {
         }
         return new JsonObject().put(CORRELATION_ID, context.correlationId()).put(SESSION_ID_KEY, context.sessionId())
                 .put(BEARER_TOKEN_KEY, context.bearerToken()).put(USER_PRINCIPAL_KEY, context.userPrincipal())
-                .put(DATA_KEY, context.data()).put(PATH_KEY, pathToJson(context.path())).toString();
+                .put(DATA_KEY, context.data()).put(RESPONSE_METADATA_KEY, context.responseData())
```

Comment:
does this work? as we have a map of Map<String, Object> we can use the new JsonObject(Map) constructor here... I don't think making a put that is a map is supported by JsonObject is it?

also same comment as for adela in the other change, see comment in the `encDetail` line at: https://github.com/SAP/neonbee/pull/165#discussion_r955231128

I think we should make it clear to the user int he JavaDoc comment of responseData(), that the only supported values of the Map<String, Object> are values which are supported by the JsonObject(Map) constructor in cluste roperation mode.

Reply:
We have been using this code put(DATA_KEY, context.data()) since years, which is also a Map<String, Object>.

Maybe we never exchanged other types than basic types and String.

Nevertheless, I added the map to Jsonobject conversion.